app.py

```diff
@@ -1,20 +1,32 @@
-from flask import Flask, render_template, request, redirect, url_for, session
+import os
+import urllib.request
+from flask import Flask, flash, render_template, request, redirect, url_for, session
 from flask_mysqldb import MySQL
 import MySQLdb.cursors
 import re
 import hashlib
+from werkzeug.utils import secure_filename
 
 app = Flask(__name__)
 
-app.secret_key = 'your useless secret key'
+app.secret_key = 'flask_key'
 
 app.config['MYSQL_HOST'] = 'localhost'
 app.config['MYSQL_USER'] = 'root'
-app.config['MYSQL_PASSWORD'] = '1234567Kd.'
-app.config['MYSQL_DB'] = 'my_db'
+
+app.config['MYSQL_PASSWORD'] = "Lowercase59%sorry%stop"
+app.config['MYSQL_DB'] = 'blogTest'
+
+UPLOAD_FOLDER = '/home/jbosch/scrapyard/3rd_try/static/img/blog'
```

Comment:
What is uploadfolder needed for?

Reply:
So that we can add images to the blog posts.